src/mode/native/mixins/update-native.js

```diff
@@ -44,12 +44,14 @@ export default {
       ) {
         const ov = this.css(dom, 'overflow') || '';
         if (/scroll|auto/.test(ov)) {
-          const { v, h } = this.getScrollProcess(dom);
+          const { v, h, isVerticalScrollable } = this.getScrollProcess(dom);
```

Comment:
Maybe we should also judge for the circumstance of horizontal scrolling?

Reply:
I was thinking about it, but I guess this is not really needed in real world. Mostly webpages are scroller vertically, so if you have a vuescroll component, that scrolls only vertically, but has same scrollWidth and clientWidth, it shouldn't be a problem, as the main document is also primary vertical scrollable, so the user will actually don't have a situation, where he tries to scroll to the right, and id would work... The user don't have a need to scroll to the right, I guess... =)
what do you think?

Reply:
I think it is necessay. We should cover all edge case.  
Image you have two scrollbars and they are inner and outer scrollbars. Inner scroll only scroll vertial and inner scroll only scroll horizontal .   And this bug will cause outer scroll could not be scrolled when your mouse is on inner scroll.

Reply:
hmm, good point. I'll add horizontal check too